docs/cudf/source/basics/basics.rst

```diff
@@ -38,46 +38,16 @@ The following table lists all of cudf types. For methods requiring dtype argumen
     +------------------------+------------------------------+-------------------------------------------------------------------------------------+---------------------------------------------+
     | Decimal                | Decimal64Dtype               | (none)                                                                              | (none)                                      |
     +------------------------+------------------------------+-------------------------------------------------------------------------------------+---------------------------------------------+
-    | List                   | ListDtype₁.₁                 | `list <https://docs.rapids.ai/api/cudf/stable/api.html#lists>`_                     | (none)₁.₂                                   |
+    | List                   | ListDtype₁.₁                 | `list <https://docs.python.org/3/library/stdtypes.html#list>`_                      | (none)₁.₂                                   |
     +------------------------+------------------------------+-------------------------------------------------------------------------------------+---------------------------------------------+
-    | Struct                 | StructDtype₁.₁               | `struct <https://nvidia.github.io/spark-rapids/docs/supported_ops.html#types>`_     | (none)₁.₂                                   |
+    | Struct                 | StructDtype₁.₁               | `struct <https://docs.python.org/3/library/stdtypes.html#dict>`_                    | (none)₁.₂                                   |
     +------------------------+------------------------------+-------------------------------------------------------------------------------------+---------------------------------------------+
 
 **Note: All dtypes above are Nullable**
 
-**1.1 The complete datatypes for both lists and structs are inferred from the data that it is composed upon. For example, the data type for the value** ``[[1, 2], [3, 4]]`` **is inferred as** ``ListDtype(int64)``.
+**1.1 The dtype of columns containing lists and structs can be inferred from the data. For example, the data type for the value** ``[[1, 2], [3, 4]]`` **is inferred as** ``ListDtype(int64)``
+**1.2 cuDF does not support string aliases for** ``list`` and ``struct`` **data types**
```

Comment:
Need an empty line between these two lines in the `.rst` if we want them to appear as separate lines in the Markdown.